docs/user/pings/testing-custom-pings.md

```diff
@@ -101,40 +101,31 @@ import my.component.GleanMetrics.CustomPingData
 import org.mockito.Mockito.spy
 import org.mockito.Mockito.`when`
 
-/**
- * This is an helper function used to enable testing mode for Glean.
- * Should only be called once before the tests, but nothing breaks if it's
- * called more than once!
- */
-fun setupGleanOnce() {
-  // Enable testing mode
-  // (Perhaps called from a @Before method so it precedes every test in the suite.)
-  Glean.enableTestingMode()
-
-  // We're using the WorkManager in a bunch of places, and Glean will crash
-  // in tests without this line. Let's simply put it here.
-  WorkManagerTestInitHelper.initializeTestWorkManager(context)
-
-  Glean.initialize(context)
-}
-
-@Test
-fun `verify custom ping metrics`() {
-  setupGleanOnce()
-
-  val scheduler = spy(MyCustomPingScheduler())
-  doAnswer {
-    // Here we validate the content that goes into the ping.
-    assertTrue(CustomPingData.sampleString.testHasValue())
-    assertEquals("test-data", CustomPingData.sampleString.testGetValue())
-
-    // We want to intercept this call, but we also want to make sure the
-    // real Glean API is called in order to clear the ping store and to provide
-    // consistent behaviour with respect to the application.
-    it.callRealMethod()
-  }.`when`(scheduler).sendPing()
-
-  scheduler.addSomeData()
-  scheduler.schedulePing()
+@RunWith(AndroidJUnit4::class)
+class MyCustomPingSchedulerTest {
+    // Apply the GleanTestRule to set up a disposable Glean instance.
+    // Please note that this clears the Glean data across tests.
+    @get:Rule
+    val gleanRule = GleanTestRule(ApplicationProvider.getApplicationContext())
```

Comment:
We should add an import for this above.

Reply:
I've intentionally left the import for this out: the package name would be different between glean-ac and glean-core, making it a bit confusing.

Reply:
Yes, but that's just temporary, right?  When we eventually release glean-core, it will be under the same namespace as glean-ac, right, so it can be a drop-in replacement?  (Or maybe that's TBD...)  In any case, it's not critical...

Reply:
That's TBD :(